scim/Dependencies.toml

```diff
@@ -5,11 +5,12 @@
 
 [ballerina]
 dependencies-toml-version = "2"
+distribution-version = "2201.5.0"
```

Comment:
As Choreo supports Ballerina `2201.4.1`,  we decided to release this connector with `2201.4.1`

Shall we delete this Dependencies.toml and build with `2201.4.1` ?
cc: @Shaaali